cookbooks/proxyfs/templates/default/usr/bin/start_and_mount_pfs.erb

```diff
@@ -134,13 +143,13 @@ if [ -f /usr/bin/systemctl ]; then
     # sudo /usr/bin/systemctl stop winbind
     sudo /usr/bin/systemctl stop proxyfsd
     await_proxyfsd_shutdown
-    sudo /usr/bin/swift-init main stop
+    sudo /usr/bin/swift-init all stop
     # sudo /usr/bin/systemctl stop swift
```

Comment:
Can I remove all lines in this section which look like "# sudo /usr/bin/systemctl stop swift" i.e. all lines that have swift commented out?

Reply:
Yes